plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/AbstractTrinoCatalog.java

```diff
@@ -284,6 +287,9 @@ protected ConnectorMaterializedViewDefinition getMaterializedViewDefinition(
     {
         ImmutableMap.Builder<String, Object> properties = ImmutableMap.builder();
         properties.put(FILE_FORMAT_PROPERTY, IcebergUtil.getFileFormat(icebergTable));
+        int formatVersion = ((BaseTable) icebergTable).operations().current().formatVersion();
```

Comment:
On latest mater we're now also missing ORC_BLOOM_FILTER_COLUMNS and ORC_BLOOM_FILTER_FPP.
I would suggest extracting common code for this from `IcebergMetadata#getTableMetadata` and re-suing it here to avoid the need to update in two places.

Reply:
Thanks for your advice! 
I have another question, does `SHOW CREATE TABLE` need to show `orc_bloom_filter_columns` and `orc_bloom_filter_fpp` properties? These two properties are specific to the orc format and are not generic, I personally think they should be shown through `$properties` instead of `SHOW CREATE TABLE`, of course $properties can already show them now.
https://trino.io/docs/current/connector/iceberg.html#properties-table

Reply:
It looks like the bloom filter properties already don't show up in SHOW CREATE if bloom filter is configured (correct me if I'm wrong). That behaviour seems ok to me.
It would be great if we also include a test in BaseIcebergMaterializedViewTest which uses the orc bloom filter properties.

Reply:
Thanks!

Currently, `SHOW CREATE TABLE` shows `orc_bloom_filter_columns ` and `orc_bloom_filter_fpp `:
<img width="803" alt="image" src="https://user-images.githubusercontent.com/29967142/170939704-fd286e97-9c43-4dd2-8fad-871a345eddb0.png">

I will add test cases in BaseIcebergMaterializedViewTest.

Reply:
@raunaqmorarka I updated the `testShowCreate()` as I felt no need to add a separate test case to test this situation.